app/helpers/audits_helper.rb

```diff
@@ -106,6 +106,26 @@ def audit_time(audit)
     date_time_absolute(audit.created_at)
   end
 
+  def audit_affected_locations(audit)
+    base = audit.locations.authorized(:view_locations)
+    authorizer = Authorizer.new(User.current, base)
+    if base.empty?
+      _('none')
+    else
+      base.map { |location| link_to_if_authorized location.name, hash_for_edit_location_path(location).merge(:auth_object => location, :permission => 'edit_locations', :authorizer => authorizer) }.to_sentence.html_safe
```

Comment:
This is a very long line, maybe break it into a `do...end` block so it is easier to read?